From 9d318888ac59ca4a1933c554c96256cbcfbd08f6 Mon Sep 17 00:00:00 2001
From: dhivael <dhivael.git@eno.space>
Date: Sat, 11 Jan 2020 15:04:04 +0100
Subject: [PATCH 12/13] rockchip: move usb0 after mmc1 in default boot order

now that we support booting from USB we should prefer USB boot over eMMC
boot. we still try to boot from SD cards first.
---
 include/configs/rockchip-common.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/rockchip-common.h b/include/configs/rockchip-common.h
index d5550a46575..21c95c75d05 100644
--- a/include/configs/rockchip-common.h
+++ b/include/configs/rockchip-common.h
@@ -14,7 +14,7 @@
 #ifndef CONFIG_XPL_BUILD
 
 #ifndef BOOT_TARGETS
-#define BOOT_TARGETS	"mmc1 mmc0 nvme scsi usb pxe dhcp spi"
+#define BOOT_TARGETS	"mmc1 usb nvme scsi mmc0 pxe dhcp spi"
 #endif
 
 #ifdef CONFIG_ARM64
-- 
2.54.0

